File: petstore-app/src/main/java/com/funtis/petstore/app/model/Tag.java
package com.funtis.petstore.app.model;

import javax.persistence.*;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by Sławomir Kluz on 12/09/2017.
 */
@Entity
@Table(name = "tags")
public class Tag extends AbstractEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column(nullable = false)
    @NotNull
    private String name;

    public Tag() {

    }

    public Tag(String name) {
        this.name = name;
    }

    public Long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: petstore-mock/src/main/scala/com/funtis/petstore/mock/model/Tag.scala
package com.funtis.petstore.mock.model

import com.funtis.commons.random.Randomizer

/**
  * Created by Sławomir Kluz on 13/10/2017.
  */
case class Tag(id: Long, name: String)

object Tag {
  def someTags() = (1 to Randomizer.Number.int(1, 5)).map(i => Tag(i, Randomizer.String.planet()))
}